src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -72,7 +72,7 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
 
   const handleAssignToMyTeam = async () => {
     try {
-      await mutateAsyncSetProjectTeam(teamAsLeadId);
+      await mutateAsyncSetProjectTeam(teamAsHeadId);
```

Comment:
we could probably just switch the team to be checked in the backend in the future